doc/nrf/gs_installing.rst

```diff
@@ -276,6 +180,34 @@ Therefore, remember to regularly check for updates:
 
          pip3 install -U west
 
+..
+
+.. _cloning_the_repositories_win:
+.. _cloning_the_repositories:
+
+.. rst-class:: numbered-step
+
+Getting the |NCS| code
+**********************
+
+The |NCS| consists of a set of Git repositories.
+
+Every |NCS| release consists of a combination of these repositories at different revisions.
+The revision of each of those repositories is determined by the current revision of the main (or manifest) repository, `sdk-nrf`_.
+
+.. note::
+   The latest state of development is on the master branch of the `sdk-nrf`_ repository.
+   To ensure a usable state, the `sdk-nrf`_ repository defines the compatible states of the other repositories.
+   However, this state is not necessarily tested.
+   For a higher degree of quality assurance, check out a tagged release.
+
+   Therefore, unless you are familiar with the development process, you should always work with a specific release of the |NCS|.
+
+The main repository, `sdk-nrf`_, contains a `west manifest file`_, :file:`west.yml`, that determines the revision of all other repositories.
+This means that sdk-nrf acts as the :ref:`manifest repository <zephyr:west-multi-repo>`, while the other repositories are project repositories.
+
+You can find additional information about the repository and development model in the :ref:`development model section <dev-model>`.
+
 Cloning the repositories
```

Comment:
Same issue as above - as numbered section, we shouldn't have subsections. And updating is not a step of installing ... 😕

Reply:
I'm going to move all sections about updating to a new section at the end of the page.